packages/oasis3-mct/package.py

```diff
@@ -215,7 +222,10 @@ def edit(self, spec, prefix):
         config = {}
 
         # TODO: https://github.com/ACCESS-NRI/ACCESS-OM/issues/12
-        NCI_OPTIM_FLAGS = "-g3 -O2 -axCORE-AVX2 -debug all -check none -traceback"
+        if "@access-esm1.5" in self.spec:
+            NCI_OPTIM_FLAGS = "-g3 -O2 -march=cascadelake -axCORE-AVX512 -debug all -check none -traceback"
```

Comment:
Is the `-march=cascadelake` actually required?

Reply:
Removed in all instances. Please check.